### PR TITLE
Add Gianluca's details to index.html

### PR DESCRIPTION
**Pull request by gianlucadetommaso**
My academic background

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Ciao Gianluca, this looks good. Can you specify if you are a PhD Student or an Enrichment Student by deleting one of them?

File: index.html [hunk 1]
```diff
@@ -513,9 +513,13 @@
         <h2>Gianluca Detommaso</h2>
         <p class="title">PhD student or Enrichment student</p>
```
Comment (amartya18x) [inline index.html]: @gianlucadetommaso  I think this is where you should mention that.

**Review by tahaceritli — DISMISSED** [review]

**Comment by martintoreilly** [comment]
@gianlucadetommaso Please can you make the change @migliris suggested and pick either Doctoral or Enrichment student in the "title" section? Thanks,.

**Review by amartya18x — CHANGES_REQUESTED** [review]

**Comment by martintoreilly** [comment]
Made requested change and tested locally so merging

**Pull request merged**